### ISSUE TITLE
Remove t.e.xmlreflector before Twisted 2.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10424]
|[<img alt="jdavisp3's avatar" src="https://avatars.githubusercontent.com/u/43582?s=50" width="50" height="50">](https://github.com/jdavisp3)| @jdavisp3 reported|
|-|-|
|Trac ID|trac#661|
|Type|defect|
|Created|2004-08-10 05:49:10Z|

<details><summary>Searchable metadata</summary>

```
trac-id__661 661
type__defect defect
reporter__davep davep
priority__highest highest
milestone__Core_2_5 Core-2.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1092116950000000 1092116950000000
changetime__1151876525000000 1151876525000000
version__ 
owner__ 
cc__davep
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jdavisp3's avatar" src="https://avatars.githubusercontent.com/u/43582?s=50" width="50" height="50">](https://github.com/jdavisp3)|@jdavisp3 commented|
|-|-|

```
#!html
<pre>
xmlreflector seems to have a bug -- maybe related to &lt; in strings
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

xmlreflector is deprecated.  I think we'll probably drop it from the next release.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

Review-ready in [branches/remove-xmlreflector-661](branches/remove-xmlreflector-661)

**Comment by twisted-trac** [comment]
|[<img alt="jyknight's avatar" src="https://avatars.githubusercontent.com/u/873903?s=50" width="50" height="50">](https://github.com/jyknight)|@jyknight set owner to @exarkun|
|-|-|

Looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17462]) Merge remove-xmlreflector-661

Author: exarkun
Reviewer: jknight
Fixes [#10424](https://github.com/twisted/twisted/issues/10424)

Remove twisted.enterprise.xmlreflector and its examples.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
